ma_connection.c

```diff
@@ -586,7 +586,11 @@ SQLRETURN MADB_DbcConnectDB(MADB_Dbc *Connection,
     MADB_Dsn *Dsn)
 {
   char StmtStr[128];
+#if defined(__s390x__)
```

Comment:
It can be just changed to my_bool without if defined. Or just initialized with someting like 0x01010101